pages/deployment/multicontainer.md

```diff
@@ -0,0 +1,131 @@
+---
+title: Coordinate services
+excerpt: A guide to building multicontainer applications with resin.io
+---
+
+# Coordinate services
+
+As your applications grow more complex, you may find significant benefit in running some services in separate containers. Splitting your application into multiple containers allows you to better isolate and maintain key services, providing a more modular and secure approach to application management. Each service can be packaged with the operating environment and tools it specifically needs to run, and each service can be limited to the minimum system resources necessary to perform its task. The benefits of multicontainer applications compound as the complexity of the application grows. Because each service can be updated independently, larger applications can be developed and maintained by separate teams, each free to work in a way that best supports their service. 
+
+
+For devices with supervisor version greater than 7.0.0, resin.io provides multicontainer application support, built around the **[Docker Compose][docker-compose]** tool. This guide will cover the considerations you need to take into account when using a multicontainer instance, including **Docker Compose** configuration, important resin.io specific settings, and single container applications.
+
+## Docker Compose
+
+The multicontainer functionality provided by resin.io is built around **[Docker Compose][docker-compose]**, an open-source tool for running multiple **Docker** containers and coordinating resources between services. The resin.io device supervisor implements a subset of the [Compose v2.1 feature set][compose-features]. You can find a full list of supported and known unsupported features in our [device supervisor reference docs][compose-support].
```

Comment:
This is kind of misleading, as I would assume from that that we're using docker's open source tool, but we're just using their file format.